packages/orderbook/test/quote/test.test.ts

```
@@ -188,17 +189,13 @@ describe('Rain Orderbook Quote Package Bindgen Tests', async function () {
-		try {
-			const result = await doQuoteTargets([target], mockServer.url + '/rpc-url');
-			const expected: OrderQuoteValue = {
-				maxOutput: '0x1',
-				ratio: '0x2'
-			};
-			assert.deepEqual(result[0], expected);
-		} catch (error) {
-			console.log(error);
-			assert.fail('expected to resolve, but failed');
-		}
+		const result = await doQuoteTargets([target], mockServer.url + '/rpc-url');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)